rgdc/rgdc/rgdc.py

```diff
@@ -119,23 +117,27 @@ def download_raster_entry_thumbnail(
 
     def download_raster_entry(
         self,
-        raster_meta_entry_id: Union[str, int],
+        raster_meta_entry_id: Union[str, int, dict],
         pathname: Optional[str] = None,
         nest_with_name: bool = False,
+        keep_existing: bool = True,
     ) -> RasterDownload:
         """
         Download the image set associated with a raster entry to disk.
 
         Args:
-            raster_meta_entry_id: The id of the RasterMetaEntry, which is a child to the desired raster entry.
+            raster_meta_entry_id: The id of the RasterMetaEntry, which is a child to the desired raster entry, or search result.
             pathname: The directory to download the image set to. If not supplied, a temporary directory will be used.
             nest_with_name: If True, nests the download within an additional directory, using the raster entry name.
+            keep_existing: If False, replace files existing on disk
```

Comment:
```suggestion
            keep_existing: If False, replace files existing on disk. Only valid if `pathname` is given.
```

Because if no `pathname`, these all go into a tempdir which is different on each call

Reply:
Just making sure all are aware of this